notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+    "        and  band  parameters  are specified on the command line it will\n",
+    "        not be possible to change them interactively.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### ( -- In multiaperture spectra go to the  spectrum  in  the  preceding         image  line.\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "In multiaperture spectra go to the  spectrum  in  the  preceding\n",
+    "        image  line.   If  there  is only one line go to the spectrum in\n",
+    "        the preceding band."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### ) -- In multiaperture spectra go to the  spectrum  in  the  following         image  line.\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "In multiaperture spectra go to the  spectrum  in  the  following\n",
+    "        image  line.   If  there  is only one line go to the spectrum in\n",
+    "        the following band."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### q -- quit and go to the next input spectrum\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Quit and go on to next input spectrum.  After the last  spectrum\n",
+    "        exit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### # -- Get  a  different  line  in   multiaperture   spectra   or   two          dimensional images.\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Get  a  different  line  in   multiaperture   spectra   or   two \n",
+    "        dimensional images.  The aperture/line/column is queried."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### % -- Get a different band in a three dimensional image.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### :dispaxis <val> -- Show or change dispersion axis for 2D images.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### :sum -- Show or change summing for 2D images.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Other implemented/to-implement SPLOT functionality"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Options: histogram\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Option \"histogram\"  plots  the  spectra  in  a  histogram  style\n",
+    "        rather  than  connecting  the  pixel  centers. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Specviz Solution\n",
+    "\n",
+    "`specviz` should default to histogram style, and it's a \"nice to have\" option to be able to toggle.  But either way this is non-critical."
+   ]
```

Comment:
I think histogram and scatter should both be options, but the default should be line-plot.  Overplotting data points and errorbars on line plots should also be an option.

Reply:
Hmm, this topic may need further discussion, but I think once both are implemented it's a simple toggle of the default.  So I won't have it hold up this PR, but I disagree that the default should be line-plot.

(And :+1: on overplotting - see the uncertainty/mask discussion I also added.)